mal/core.py

```diff
@@ -50,9 +50,16 @@ def start_end(entry, episode, total_episodes):
         entry['status'] = MyAnimeList.status_codes['completed']
         entry['date_finish'] = date.today().strftime('%m%d%Y')
         print(color.colorize('Series completed!', 'green'))
-        score = int(input('Enter a score (or 0 for no score): '))
-        if score != 0:
-            entry['score'] = score
+
+        # set/change score
+        user_score = input(
+            'Enter new score (leave blank to keep score at {}): '.format(
+            entry.get('score', 0))
+        ).strip()
+        if user_score: # do nothing if blank answer
+            try: entry['score'] = int(user_score)
```

Comment:
Please, keep the PEP8 format, break the lines.

Reply:
done